src/SigasiProjectCreator/convertDotFtoCsv.py

```diff
@@ -12,9 +12,16 @@
 import os
 import csv
 
+def isabspath(path):
+    s_path = str(path)
+    if s_path.startswith('\\') or s_path.startswith('/') or s_path[1] == ':' or s_path.startswith('$'):
+        return True
+    return os.path.isabs(path)
```

Comment:
Duplication, also should just be using os.path.isabs (and expand paths before checking this). The `$` might expand to a relative path.

Reply:
This script will be removed in a different PR soon, so duplication will be fixed